platform-sdk/swirlds-benchmarks/src/jmh/java/com/swirlds/benchmark/CryptoBench.java

```diff
@@ -212,19 +218,25 @@ public void transferPrefetch() throws Exception {
 
         initializeFixedAccounts(virtualMap);
 
+        final List<VirtualMap<BenchmarkKey, BenchmarkValue>> toRelease = new ArrayList<>();
+
         long prevTime = System.currentTimeMillis();
         for (int i = 1; i <= numFiles; ++i) {
+            toRelease.add(virtualMap);
+
             // Generate a new set of unique random keys
             final Integer[] keys = generateKeySet();
 
             // Warm keys in parallel asynchronously
             final VirtualMap<BenchmarkKey, BenchmarkValue> currentMap = virtualMap;
-            for (int thread = 0; thread < numThreads; ++thread) {
-                final int idx = thread;
+            for (int j = 0; j < keys.length; j += KEYS_PER_RECORD) {
+                final int key = j;
                 prefetchPool.execute(() -> {
-                    for (int j = idx * KEYS_PER_RECORD; j < keys.length; j += numThreads * KEYS_PER_RECORD) {
-                        currentMap.warm(new BenchmarkKey(keys[j]));
-                        currentMap.warm(new BenchmarkKey(keys[j + 1]));
+                    try {
+                        currentMap.warm(new BenchmarkKey(keys[key]));
+                        currentMap.warm(new BenchmarkKey(keys[key + 1]));
+                    } catch (final Exception e) {
+                        logger.error("Warmup exception", e);
```

Comment:
Does it makes sense to continue the benchmark in such case?

Reply:
Yes. We have #5383 and it's good to know if it happens (and how many times). It has no effect on functionality, though, and negligible effect on performance (if rare). Unconditionally breaking a many-hour run because of that would be frustrating. Rather, the error is reported and it's up to the operator to decide whether the run should be stopped.